config.json

```diff
@@ -604,7 +604,23 @@
         "Control flow (loops)",
         "Strings"
       ]
+    },
+    {
+      "uuid": "6ac4ad5f-a64a-4646-91c5-169d53f289f9",
+      "slug": "run-length-encoding",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "Control flow (conditionals)",
+        "Exception handling",
+        "Parsing",
+        "Text formatting",
+        "Regular expressions",
+        "Pattern recognition",
+        "Strings"
+      ]
```

Comment:
Will  add additional test cases per the  canonical test file and will squash the commits into on. Will also update the exercise order.